clients/src/main/java/org/apache/kafka/clients/consumer/internals/OffsetFetcherUtils.java

```diff
@@ -315,6 +319,57 @@ void onFailedRequestForResettingPositions(
             log.error("Discarding error in ListOffsetResponse because another error is pending", error);
     }
 
+
+    void onSuccessfulRequestForValidatingPositions(
```

Comment:
it should be onSuccessfulResponse ? As well as onFailedResponse.  Maybe `onSuccessfulValidatePositionResponse`